certification-content-bundle-spec.md

```diff
@@ -0,0 +1,118 @@
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it easier to see the general file structure.
+
+```yml
+entity_type: Certification
+schema_version: 1
+
+default_locale: en
+
+title: 
+  locales:
+    en: Virtual Machines in GCP
+
+description:
+  locales:
+    en: Prove what you know about VMs in GCP by doing these courses and exams!
+
+objectives:
+  locales:
+    en:
+      - 1st learning objective
+      - 2nd learning objective
+      - 3rd learning objective
+      - etc...
+
+audience:
+  locales:
+    en: >
+      This certification is intended for people who want to test their knowledge
+      of VMs in GCP.
+
+prerequisites:
+  locales:
+    en: >
+      It is recommended to know a little bit about Unix operating systems before
+      beginning this certification.
+
+tags:
+product_tags:
+role_tags:
+domain_tags:
+
+credits: 80
+
+certificate_award: qwiklabs-accredible
+
+steps: ...
+```
+
+### Certification attributes
+
+attribute          | required | type       | notes
+-------------------| -------- | ---------- | -----------------------------------------
+default_locale     | ✓        | string     | Corresponds to the locale that the certification is authored in. Authoring tools can use this as a hint to notify localizers when content in the default locale is updated. Also, it provides a hint to the learner interface about which locale to display if an instruction/resource is not localized for the learner's current locale.
+schema_version     | ✓        | integer    | Which version of the certification bundle schema you are using
+title              | ✓        | dictionary | A locale dictionary of the certification title, such as "Virtual Machines in GCP"
+description        |          | dictionary | A locale dictionary of the certification description which describes the contents of the certification
+objectives         |          | dictionary | A locale dictionary of objective arrays
+audience           |          | dictionary | A locale dictionary of audiences
+prerequisites      |          | dictionary | A locale dictionary of prerequisites
+tags               |          | array      | Array of strings to be used as hints in searching, etc.
+product_tags       |          | array      | Array of strings from the "Products" column in [this sheet](https://docs.google.com/spreadsheets/d/1hUUch85HBRsRJsgRo9VCg0Pn7ZXi21sl6JU7VOr9LP8)
+role_tags          |          | array      | Array of strings from the "Roles" column in [this sheet](https://docs.google.com/spreadsheets/d/1hUUch85HBRsRJsgRo9VCg0Pn7ZXi21sl6JU7VOr9LP8)
+domain_tags        |          | array      | Array of strings from the "Domain" column in [this sheet](https://docs.google.com/spreadsheets/d/1hUUch85HBRsRJsgRo9VCg0Pn7ZXi21sl6JU7VOr9LP8)
+credits            |          | integer    | Price of the certification
+certificate_award  | ✓        | string     | A unique identifier for the corresponding award that this certification grants
```

Comment:
Where are awards defined?

Reply:
The way I have things now, they would need to be an entirely new and different spec.

Reply:
We've decided to put this off until later. Removing certificate_award from this PR

Reply:
@carinetorres is working on the specifics of this